input/pagecontent/group-genomics.md

```diff
@@ -1,21 +1,94 @@
 
-* [GenomicsReport] - Contains results of genomic analyses. Genomic reports vary in complexity and content, as simple as the results for a single discrete variant to complex sequences. This profile inherits from the profile of the same name in CGRIG.
-* [GenomicVariant] - Used to record variants that could be found from tests that broadly analyze genomic regions (e.g.: exome tests) and stores results for any variants that could have been found. The region in which the variant was found could be specified in the RegionStudied attribute of the GenomicsReport profile. This profile inherits from the Variant profile in CGRIG.
-* [GenomicRegionStudied] - Used to record the portion(s) of the genome that was tested for variants. This profile inherits from the RegionStudied profile in CGRIG.
+Three profiles relate to the capture of clinical genomics data:
+
+* [GenomicsReport] - Contains results of genomic analyses. Genomic reports vary in complexity and content, as simple as the results for a single discrete variant to complex sequences. This profile inherits from the profile of the same name in GRIG.
+* [GenomicVariant] - Used to record variants that could be found from tests that broadly analyze genomic regions (e.g.: exome tests) and stores results for any variants that could have been found. The region in which the variant was found could be specified in the RegionStudied attribute of the GenomicsReport profile. This profile derives from the Variant profile in GRIG.
+* [GenomicRegionStudied] - Used to record the portion(s) of the genome that was tested for variants. This profile derives from the RegionStudied profile in GRIG.
+
+Additionally, the additional GRIG profiles are included in the mCODE bundle:
+
+* [Diagnostic Implication](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition-diagnostic-implication.html)
+* [Therapeutic Implication](http://hl7.org/fhir/uv/genomics-reporting/STU2/StructureDefinition-therapeutic-implication.html)
+* [Microsatellite Instability (MSI)](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/msi)
+* [Tumor Mutation Burden (TMB)](http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/tmb)
+
+The diagram below illustrates the distinction within an mCODE bundle.
+
+<div style="text-align: center;">
+<img src="mcode-cg-overlap.svg" alt="mCODE use of GRIG profiles" />
+</div>>
+
+<br/>
+
+The mCODE example [`Bundle-gx-genomic-bundle-incredible-hulk`](Bundle-gx-genomic-bundle-incredible-hulk.html) demonstrates a bundle containing resources conforming with the GRIG genomics report.
```

Comment:
Re: the fictional character - yeh, I wanted to accurately represent what was in the test genomics report that was provided by the reference lab and that was the patient name they chose. Originally, I had a PDF attachment as base64 encoded in the example and it would not have matched if an implementer processed it. 
Re: the formatting - yeh, I think it is a publisher bug but I'll check just in case and confirm.